db_test.go

```diff
@@ -214,31 +213,31 @@ func TestDB_DeleteTracker(t *testing.T) {
 		txn.Commit()
 	}
 
-	assert.EqualValues(t, table.Revision(db.ReadTxn()), expvarInt(metrics.RevisionVar.Get("test")), "Revision")
-	assert.EqualValues(t, 3, expvarInt(metrics.ObjectCountVar.Get("test")), "ObjectCount")
-	assert.EqualValues(t, 0, expvarInt(metrics.GraveyardObjectCountVar.Get("test")), "GraveyardObjectCount")
```

Comment:
These changes are examples of alecthomas/assert's type safety. Previously strechr/testify was comparing `int64`s and `uint64`s.